website/blog/2025-10-08-react-native-0.82.md

```diff
@@ -0,0 +1,317 @@
+
+<Tabs>
+  <TabItem label="yarn" value="yarn" default>
+    ```
+    "resolutions": {
+        "hermes-compiler": "250829098.0.1"
+    }
+    ```
+  </TabItem>
+  <TabItem value="npm" label="npm">
+    ```
+    "overrides": {
+        "hermes-compiler": "250829098.0.1"
+    }
+    ```
+  </TabItem>
+</Tabs>
+
+2. Enable Hermes V1 for Android by adding `hermesV1Enabled=true` inside the `android/gradle.properties` file, and configure React Native [to build from source](https://reactnative.dev/contributing/how-to-build-from-source#android) by editing `android/settings.gradle`:
+
+<Tabs>
+  <TabItem value="android/gradle.properties" label="android/gradle.properties" default>
+    ```diff
+    + hermesV1Enabled=true
+    ```
+  </TabItem>
+  <TabItem label="android/settings.gradle" value="android/settings.gradle">
+    ```diff
+    + includeBuild('../node_modules/react-native') {
+    +    dependencySubstitution {
+    +        substitute(module("com.facebook.react:react-android")).using(project(":packages:react-native:ReactAndroid"))
+    +        substitute(module("com.facebook.react:react-native")).using(project(":packages:react-native:ReactAndroid"))
+    +        substitute(project(":packages:react-native:ReactAndroid:hermes-engine")).using(module("com.facebook.hermes:hermes-android:250829098.0.1"))
+    +    }
+    + }
+    ```
+  </TabItem>
+</Tabs>
+
+3. Enable Hermes V1 for iOS by installing pods with `RCT_HERMES_V1_ENABLED=1` environment variable. Keep in mind that Hermes V1 is not compatible with the precompiled React Native builds, so make sure you don’t use the `RCT_USE_PREBUILT_RNCORE` flag when installing pods:
+
+```sh
+RCT_HERMES_V1_ENABLED=1 bundle exec pod install
+```
+
+## React 19.1.1
+
+This release of React Native ships with the latest React stable: [React 19.1.1](https://github.com/facebook/react/releases/tag/v19.1.1).
+
+This release of React contains full support for owner stacks for React Native. Back in React Native 0.80, when we shipped support for 19.1.0, we [mentioned](/blog/2025/06/12/react-native-0.80#react-1910) that owner stacks were not fully supported if you were using the [`@babel/plugin-transform-function-name`](https://babeljs.io/docs/babel-plugin-transform-function-name) Babel plugin. This release lifts this restriction and enables owner stacks for all React Native users.
+
+| BEFORE                                                                                           | AFTER                                                                                        |
+| ------------------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------------- |
+| ![Example error thrown without Owner Stacks](../static/blog/assets/0.82-owners-stack-before.png) | ![Example error thrown with Owner Stacks](../static/blog/assets/0.82-owners-stack-after.png) |
```

Comment:
This should center the images within the column and perhaps make them a bit smaller.
If it is not `center` is `centre`, I never remember what's the right one here.

```suggestion
| <center><img alt="Example error thrown without Owner Stacks" src="../static/blog/assets/0.82-owners-stack-before.png" height=480 /></center> | <center><img alt="Example error thrown with Owner Stacks" src="../static/blog/assets/0.82-owners-stack-after.png" height=480 /></center> |
```